src/Projection/PdoEventStoreProjector.php

```diff
@@ -164,6 +170,7 @@ public function __construct(
         $this->persistBlockSize = $persistBlockSize;
         $this->sleep = $sleep;
         $this->status = ProjectionStatus::IDLE();
+        $this->triggerPcntlSignalDispatch = extension_loaded('pcntl') && $triggerPcntlSignalDispatch;
```

Comment:
switch the order of the checks so that if `$triggerPcntlSignalDispatch = false` PHP does not check if extension is loaded